app/src/pages/index.tsx

```diff
@@ -1,125 +0,0 @@
-import Head from 'next/head'
```

Comment:
For this file and the previous I'm just experiencing some git weirdness. I moved the app to `next-app` to disambiguate it from `vite-app` but something obviously went wrong in the process.